src/pages/billing/CreateNewBillingProjectWizard.test.js

```diff
@@ -167,6 +170,9 @@ describe('CreateNewBillingProjectWizard Steps', () => {
     it('has the correct initial state for Step 4', () => {
       testStep4Disabled()
     })
+    it('passes accessibility checks in initial state', async () => {
```

Comment:
@yunip20 added some smoke testing for accessibility. This is how the test fails when there is an issue (I temporarily changed one of the `h3` elements to an `h5`).

![image](https://user-images.githubusercontent.com/484484/198624074-c6bef5ef-fe86-4c94-9cac-7ba3de43fe24.png)